.github/workflows/helm-chart.yml

```diff
@@ -15,15 +13,15 @@ permissions:
   pull-requests: read
 
 jobs:
-  release:
+  publish-helm:
     runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           submodules: true
           fetch-depth: 0
+          ref: ${{ github.event.client_payload.tag }}
 
       - name: Publish Workspace Helm chart
```

Comment:
Uploads helm chart to GH pages right? I can't see our charts here yet: https://azure.github.io/kaito/